driver-core/src/test/functional/com/mongodb/ClusterFixture.java

```diff
@@ -514,6 +520,11 @@ public static boolean isAuthenticated() {
         return getConnectionString().getCredential() != null;
     }
 
+    public static boolean isClientSideEncryptionTest() {
```

Comment:
Client side encryption tests aren't run since it's all client side, but running the tests exposed the fact that a few of the FLE tests don't check for these system properties.  Rather than copy this little block of code to yet another place, I added this little helper and replaced the block in existing tests as well as added it to ones where the check was missing.